t/20_sdaf_deployment_library.t

```diff
@@ -640,4 +659,30 @@ subtest '[sdaf_execute_playbook] Command verbosity' => sub {
     undef_variables();
 };
 
+subtest '[git_clone] Compose command' => sub {
+    my $ms_sdaf = Test::MockModule->new('sles4sap::sdaf_deployment_library', no_auto => 1);
+    my @calls;
+    $ms_sdaf->redefine(assert_script_run => sub { @calls = $_[0]; return 0; });
+    $ms_sdaf->redefine(record_info => sub { return 0; });
+    my $repository = 'https://github.com/os-autoinst/os-autoinst-distri-opensuse';
+
+    git_clone($repository);
+    is join(' ', @calls), "git clone $repository", "Check base command composition";
+
+    git_clone($repository, skip_ssl_verification => 'true', output_log_file => 'output.log', branch => 'world_domination');
+    note("\n  -->  " . join("\n  -->  ", @calls));
+    ok(grep(/git -c http.sslVerify=false clone/, @calls), 'Option "-c http.sslVerify=false " must be between "git" and "clone"');
```

Comment:
something generic about UT, "Do not test too strictly" : practically speaking if you test for

```
'Option "-c http.sslVerify=false " must be between "git" and "clone"'
```

regexp can be maybe just `git.*-c http\.sslVerify=.*clone`

Reply:
Oh yes, thanks. You are right. Fixed,